autoload/neoterm/repl/python.vim

```diff
@@ -0,0 +1,63 @@
+"init python specific settings
+if !exists('g:neoterm_repl_ipython_magic_paste')
+  let g:neoterm_repl_ipython_magic_paste = 0
+end
+
+function! neoterm#repl#python#is_valid(value)
+  let l:cmd =  type(a:value) == v:t_list ? join(a:value) : a:value
+
+  if l:cmd =~# '\<ipython\>'
+    return executable('ipython')
+  elseif l:cmd =~# '\<python\>'
+    return executable('python')
+  else
+    return v:false
+  end
+endfunction
+
+function! s:ipython_magic_command_for(command)
```

Comment:
can you put the private functions after the public function that call them? In this case, after the  `neoterm#repl#python#exec`